### ISSUE TITLE
Use GitHub actions for frictionless automated releases

### ISSUE THREAD

**Issue by Atmosfearful** [KlimaDAO/klimadao#206]
# Goal
- A robot merges `staging` to `main` automatically on Mondays , Tuesdays, Wednesdays and Thursdays

Stretch Goals
- Merge & rebase the translators branch. We can do this manually to start to test out the workflow.
- Publish some form of changelog to the public discord server so the community can pat us on the back with 🎉 emojis @0xAurelius 

### But what about QA and translations?
- A PR should never be merged to staging unless we are confident it's production-ready.
- PRs with big or critical changes should get extra testing from QA team before being merged to staging.
- It's OK to release untranslated content. With a frequent release schedule, translators can follow-up quickly.
  - We should never have devs get blocked by translation

# Requirements

- [ ] Extract Strings and commit to staging
- [ ] Fast-forward merge of staging to main `git merge --ff staging`. Vercel does the rest.
  - needs to be a local `--ff` merge so the commit hashes match. Otherwise we end up having to constantly rebase all the downstream branches.
- [ ] Runs on a [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), I prefer early morning PST
